aten/src/ATen/dlpack.h

```diff
@@ -13,7 +13,7 @@
 #endif
 
 /*! \brief The current version of dlpack */
-#define DLPACK_VERSION 010
+#define DLPACK_VERSION 040
```

Comment:
Will there be backward-compatibility issues with this change?

For example, if another framework was using the previous version and a PyTorch program was working with that framework, will it stop working with it now?

Reply:
I think it should be safe because dlpack ABI is backward compatible, so there shouldn't be a problem. But, I think that @rgommers might have a better idea of this

Reply:
https://github.com/pytorch/pytorch/issues/55090#issuecomment-812028946 confirms that there were no ABI breaking changes. There are no other backwards incompatible changes either.